lib/factgrp.gi

```diff
@@ -1160,6 +1230,86 @@ function(G,N)
   return fail;
 end);
 
+BindGlobal("FactPermRepMaxDesc",function(g,n,maxlev)
+local lim,deg,all,c,recurse,use;
+  if ValueOption("infactorpermrep")=true then return false;fi;
+  lim:=RootInt(IndexNC(g,n),3);
```

Comment:
This line of code is not tested by the Travis test suite, i.e., the function never gets beyond the `return false` one line above.

Reply:
I'm puzzled that it ever gets even to the first line. The option tested is to avoid infinite recursion and is only set by this function. That is, the function is never called by the tiny test examples. Is supect codecov got confused.